validators/src/unit-test/java/uk/gov/moj/sdt/validators/IndividualRequestValidatorTest.java

```diff
@@ -112,23 +107,21 @@ public class IndividualRequestValidatorTest extends AbstractValidatorUnitTest {
     /**
      * Data retention period.
      */
-    private int dataRetentionPeriod = 90;
+    private static final int DATA_RETENTION_PERIOD = 90;
 
     /**
      * Setup of the Validator and Domain class instance.
      */
-    public void setUpLocalTests() {
+    @BeforeEach
+    public void setUp() {

```

Comment:
Should this method contain MockitoAnnotations.openMocks(this); to initialise the mocked objects?

Reply:
It appears that annotating the class with @ExtendWith(MockitoExtension.class) automatically sets up the mocks, so this isn't needed.